tools/agat/macros.xml

```diff
@@ -0,0 +1,41 @@
+<macros>
+    <token name="@TOOL_VERSION@">1.1.0</token>
+    <token name="@VERSION_SUFFIX@">0</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="@TOOL_VERSION@">agat</requirement>
+            <requirement type="package" version="0.34">perl-statistics-r</requirement>
+        </requirements>
+    </xml>
+    <xml name="biotools">
+        <xrefs>
+            <xref type="bio.tools">agat</xref>
+        </xrefs>
+    </xml>
+    <xml name="citations">
+        <citations>
+            <citation type="doi">10.5281/zenodo.3552717</citation>
+        </citations>
+    </xml>
+    <xml name="ANNOTATION_INPUT" token_format="gff,gtf,gff3,gff3.gz">
+        <param argument="--gff" type="data" format="@FORMAT@" label="Annotation file" help="Input GTF/GFF file" />
+    </xml>
+    <xml name="REFERENCE_FASTA">
+        <param argument="--fasta" type="data" format="fasta" label="Reference FASTA file" help="Input FASTA file" />
```

Comment:
should that not use the all_fasta loc file as well?

Reply:
Sorry, I do not understand what do you mean.

Reply:
Its a reference genome. Should it not be use the Galaxy shipped reference genomes - at least optionally?

Reply:
> Its a reference genome. Should it not be use the Galaxy shipped reference genomes - at least optionally?

Ah, yes, I'll include also the references from the databases.